pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.black]
-target-version = ["py37", "py38"]
+target-version = ["py38"]
```

Comment:
```suggestion
target-version = ["py38", "py39"]
```

Reply:
Doesn't work, black supports only up to py38 at the moment.